components/header/header.js

```diff
@@ -0,0 +1,19 @@
+import Link from 'next/link';
+import SiteLogo from '../siteLogo';
+import styles from './header.module.scss';
+
+export default function Header() {
+  return (
+    <div className={styles.header}>
+      <Link href="/">
+        <a>
+          <SiteLogo className={styles.siteLogo} />
```

Comment:
Just a thought, have we thought of a good naming convention for the styles, i feel like it could get quite convoluted and confusing

Reply:
Rn each component will have its own style sheet, so the more we divide stuff into components, the easier the naming will be.

Hopefully thanks to this our names can stay relatively descriptive & specific